packages/wallets/src/features/cfd/screens/CompareAccounts/CompareAccountsScreen.tsx

```diff
@@ -20,17 +20,21 @@ const CompareAccountsScreen = () => {
             <CompareAccountsHeader isDemo={isDemo} isEuRegion={isEuRegion} />
             <div className='wallets-compare-accounts__card-list'>
                 <CompareAccountsCarousel>
-                    {mt5Accounts?.map(item => (
-                        <CompareAccountsCard
-                            isDemo={isDemo}
-                            isEuRegion={isEuRegion}
-                            isEuUser={isEuUser}
-                            key={`${item?.market_type} ${item?.shortcode}`}
-                            marketType={item?.market_type}
-                            platform={item?.platform}
-                            shortCode={item?.shortcode}
-                        />
-                    ))}
+                    {mt5Accounts?.map(
+                        item =>
+                            // @ts-ignore
```

Comment:
`api-types` version needs to be updated so that we receive products as one of the types. I updated it and faced some issues after the update which are out of scope of this PR.